rust/private/clippy.bzl

```diff
@@ -75,9 +81,13 @@ def _clippy_aspect_impl(target, ctx):
         build_info,
     )
 
-    # A marker file indicating clippy has executed successfully.
-    # This file is necessary because "ctx.actions.run" mandates an output.
-    clippy_marker = ctx.actions.declare_file(ctx.label.name + ".clippy.ok")
+    capture_output = ctx.attr._capture_output[ClippyConfigInfo].capture_output
```

Comment:
Could you please move this block below after `construct_arguments`, and merge all 3 `if capture_output` blocks into one? I would move `--test` above clippy flags, below `construct_arguments`

Reply:
Done. I did it that way to minimize the diff, but now that's clear, your suggestion is better.